llvm/lib/Transforms/IPO/SampleProfileProbe.cpp

```diff
@@ -221,12 +222,26 @@ void SampleProfileProber::computeProbeIdForBlocks() {
 }
 
 void SampleProfileProber::computeProbeIdForCallsites() {
+  LLVMContext &Ctx = F->getContext();
+  Module *M = F->getParent();
+
   for (auto &BB : *F) {
     for (auto &I : BB) {
       if (!isa<CallBase>(I))
         continue;
       if (isa<IntrinsicInst>(&I))
         continue;
+
+      // There's a limit on the number of probe indices that can be optimized.
```

Comment:
nit: remove the firs sentence in the comment, as the 2nd sentence should be sufficient. "that can be optimized" in particular is a bit unclear.

Reply:
changed.